paper/symposium.md

```diff
@@ -0,0 +1,214 @@
+
+*Patricia Martin, Doctoral Student at UCLA* and investigator with the
+Enrollment Management, Recruiting and Access Group, will provide an
+overview of the Unrollment Project, including both the substantive
+goals of the project and the process for undertaking research. The
+Unrollment project has an unusual structure for higher education
+research, where all data and code are publicly available, and research
+can be followed in "real time" as researchers post their results and
+discussions to GitHub, an open repository for code.  The project
+itself has several current key milestones, including the development
+of predictive algorithms for college completion; simulation of
+decision-making based on these algorithms; and calculating likely
+impacts for different groups of students based on these
+milestones. Martin will introduce the use of git and GitHub as a
+model for open-source research collaborations that are available to
+the public in development, and will demonstrate how audience members
+can provide input and feedback to the research team as the work
+progresses. 
+
+*Benjamin Skinner, Assistant Professor of Higher Education and Policy* at
+the University of Florida, will discuss the results of the first stage
+of the project, which involves generating predictive algorithms for
+bachelor's degree completion among recent high school graduates
+attending four-year institutions. The data for this part of the study
+come from the Educational Longitudinal Study of 2002 (Ingels, et al
+2014). Using this data, the project predicts college graduation based
+on a large set of covariates, using four different modeling
+approaches: logistic regression, elastic-net logistic regression,
+random forests and neural nets (Hastie et al, 2009). Skinner will
+summarize the results of these models, including both standard
+measures of model fit and summaries of the substantive
+importance of these different measures of fit.   
+
+*Monique Ositelu, Senior Policy Analyst at the New America Foundation*,
+will discuss her work on the implications of using predictive
+algorithms for individual institutions. These case studies of
+institutional decision-making will provide concrete examples of how
+predictive algorithms influence administrators' decision-making and
+thereby impacting students' chances of success.  This analysis
+evaluates the equity impacts of predictive analytics on enrollment
+management within higher education. In particular, we evaluate the
+unintended consequences of predictive analytics and implications on
+access for students of color and low-income students. We identify how
+sectors of higher education institutions are differentially affected
+by using algorithms for predicting enrollment and graduation. We
+identify which sectors reinforce existing inequities and limit access
+to higher education for underserved populations.
+
+The sectors we will evaluate are 4-year public and 4-year private
```

Comment:
I found this paragraph and the one below oddly placed as within the description of panelists. seems like this would go prior to description of panelists.